sources/packages/web/package.json

```diff
@@ -34,7 +34,7 @@
     "vue": "^3.2.31",
     "vue-class-component": "^8.0.0-0",
     "vue-router": "^4.3.2",
-    "vuetify": "^3.4.2",
+    "vuetify": "^3.7.2",
```

Comment:
The alerts seem to be with extra spacing on the top.
![image](https://github.com/user-attachments/assets/c5ee647e-df8e-4814-b163-24bf6f672537)

Reply:
Fixed